lms/services/event.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
```

Comment:
This makes the `if TYPE_CHECKING` work as expected. 

Allow a type annotation reference a non yet imported name.